app/controllers/application_controller.rb

```diff
@@ -23,7 +23,7 @@ def index
   # @param [String] action The action to render when no object passed.
   #   (Creation / update failure, destruction success)
   # @param [String] path The path to redirect to when no object passed.
-  def handle_action(object:, msg:, action: nil, path: nil)
+  def handle_action(object:, msg:, action: nil, path: nil, **_)
```

Comment:
The method gets passed the splatted `results` from creator objects; this allows those results to have arbitrary keys that get used in the actual controller method (here, `:group` is the key that i used)